content/04.results.md

```diff
@@ -2,42 +2,54 @@
+The Compound treats Disease (CtD) edge type is a quintessential example of this trend.
+The baseline model starts off with an AUROC score of 52% and an AUPRC of 29%, then performance increases to 76% and 48% respectively as more CtD related label functions are included. 
+Gene interacts Gene (GiG), Disease associates Gene (DaG) and Compound binds Gene (CbG) edge types share a similar trend.
+Baseline performance starts off with an AUROC of 57% for DaG, 78% for CbG and 66% for GiG, then increases to 62% for DaG, 81% for CbG and 75% for GiG.
+Similarly with AUPR, the baseline starts off at 43% for DaG, 29% for CbG and 30% for GiG, then increases to 44%, 30% and 35% respectively.
+
+We based our initial hypothesis on the idea that certain edge types would share related linguistic features and terminology which would be amenable to label function transfer.
+For example, sentences for edges CbG and GiG typically describe physical interactions whether it be between a compound and a gene or a pair of two protein produced from genes.
+We observed performance increases in terms of both AUROC and AUPR when using label functions from the GiG edge type to predict CbG edges (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}).
+Likewise when using CbG related label function to predict the GiG edge type; however, in both cases GiG related label functions out performed the CbG related label functions.
+Regarding DaG, each label function source had similar performance when predicting the test set (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}); however the difference is more apparent when predicting the tune set (Supplemental Figures {@fig:auroc_gen_model_tune_set} and {@fig:auroc_gen_model_tune_set}).
+
+We pooled together all non-database label functions for each edge type.
+We randomly sampled label functions from the combined pool, then trained a generative model to predict each edge type.
+Performance is report in terms of AUROC and AUPR (Figures {@fig:auroc_grabbag_gen_model_test_set} and {@fig:aupr_grabbag_gen_model_test_set}; Supplemental Figures @fig:auroc_grabbag_gen_model_tune_set} and @fig:aupr_grabbag_gen_model_tune_set}).
+General consensus for CbG, GiG and CtD edges is that using all label functions hurt performance.
+The gap between edge specific sources and the all category widens as the number of sampled label functions increases.
+The only exception to this is DaG where performance increases (57% to 65% AUROC and 43% to 47% AUPR) as more and more label functions are added to the baseline model.
+
+Overall, only CbG and GiG show significant signs of transferability between each other for both Tune and Test set.
+DaG shows conflicting results between the tune and test set, when it comes to sampling individual off-edge label functions; however, when pooling every label function together performance increases. 
```

Comment:
I think that when you see a difference between tune and test you have to trust test, right? We don't expect tune to be an accurate estimate of performance.